tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py

```diff
@@ -0,0 +1,119 @@
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+
+# Create an Expectation
+expectation_configuration = ExpectationConfiguration(
+    # Name of expectation type being added
+    expectation_type="expect_table_columns_to_match_ordered_list",
+    # These are the arguments of the expectation
+    # The keys allowed in the dictionary are Parameters and
+    # Keyword Arguments of this Expectation Type
+    kwargs={
+        "column_list": [
+            "account_id",
+            "user_id",
+            "transaction_id",
+            "transaction_type",
+            "transaction_amt_usd",
+        ]
+    },
+    # This is how you can optionally add a comment about this expectation.
+    # It will be rendered in Data Docs.
+    # See this guide for details:
+    # `How to add comments to Expectations and display them in Data Docs`.
+    meta={
+        "notes": {
+            "format": "markdown",
+            "content": "Some clever comment about this expectation. **Markdown** `Supported`",
+        }
+    },
+)
+# Add the Expectation to the suite
+suite.add_expectation(expectation_configuration=expectation_configuration)
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py create_expectation_2">
+expectation_configuration = ExpectationConfiguration(
+    expectation_type="expect_column_values_to_be_in_set",
+    kwargs={
+        "column": "transaction_type",
+        "value_set": ["purchase", "refund", "upgrade"],
+    },
+    # Note optional comments omitted
+)
+suite.add_expectation(expectation_configuration=expectation_configuration)
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py create_expectation_3">
+expectation_configuration = ExpectationConfiguration(
+    expectation_type="expect_column_values_to_not_be_null",
+    kwargs={
+        "column": "account_id",
+        "mostly": 1.0,
+    },
+    meta={
+        "notes": {
+            "format": "markdown",
+            "content": "Some clever comment about this expectation. **Markdown** `Supported`",
+        }
+    },
+)
+suite.add_expectation(expectation_configuration=expectation_configuration)
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py create_expectation_4">
+expectation_configuration = ExpectationConfiguration(
+    expectation_type="expect_column_values_to_not_be_null",
+    kwargs={
+        "column": "user_id",
+        "mostly": 0.75,
+    },
+    meta={
+        "notes": {
+            "format": "markdown",
+            "content": "Some clever comment about this expectation. **Markdown** `Supported`",
+        }
+    },
+)
+suite.add_expectation(expectation_configuration=expectation_configuration)
+# </snippet>
+
```

Comment:
How about an assertion about the content of the expectation suite after adding these expectations? I think that would be valuable to make sure that the suite contains all the expectations we added.

Reply:
great idea. Do you mean in the script only, or in the guide too?

Reply:
Good question! Just in the script so it's not just a smoke test. With this assert we will know that as long as this test passes that someone following the guide will get the end result we expect.